packages/docs/src/components/content/ContentRenderer.tsx

```diff
@@ -66,13 +68,16 @@ const TextWithMaxWidth = (props: any, Component) => {
  * A mapping of custom components for mdx syntax
  * Each mapping has a key with the element name and a value of a functional component to be used for that element
  */
-const customComponents = (theme) => ({
+const customComponents = (frontmatter, theme) => ({
   ButtonMigrationTable: (props) => <ButtonMigrationTable theme={theme} {...props} />,
   ButtonVariationsTable: (props) => <ButtonVariationsTable theme={theme} {...props} />,
   code: CodeWithSyntaxHighlighting,
   ColorExampleList: (props) => <ColorExampleList theme={theme} {...props} />,
   ColorRamps,
   ComponentThemeOptions: (props) => <ComponentThemeOptions theme={theme} {...props} />,
+  DesignResourceLink: (props) => (
+    <DesignResourceLink frontmatter={frontmatter} theme={theme} {...props} />
```

Comment:
`frontmatter` needs to be passed to `DesignResourceLink` so this component can access content that exists in the page's frontmatter (e.g., Sketch links)